draft-ietf-cose-cbor-encoded-cert.md

```diff
@@ -591,14 +591,11 @@ The formatting and processing for c5b, c5c, c5t, and c5u, defined in {{iana-head
 The COSE_C509 structure used in c5b, c5c, and c5u is defined as:
 
 ~~~~~~~~~~~ cddl
-COSE_C509 = C509CertData / [ 2* C509CertData ]
-C509CertData = bytes .cborseq C509Certificate
+COSE_C509 = C509Certificate / [ 2* C509Certificate ]
```

Comment:
I still think that the COSE embedding needs to isolate the certificate contents similar to
```
C509CertData = bytes .cbor C509Certificate
```

This adds some redundancy to the file form but the bstr isolation really helps the COSE and similar embedders.

Reply:
I updated accordingly and also added back examples at the end.